client/web/compose/src/components/ModuleFields/Configurator/validation.vue

```diff
@@ -116,6 +116,78 @@
         {{ $t('validators.description') }}
       </b-form-text>
     </b-form-group>
+
+    <div>
+      <div>
+        <h5>{{ $t('constraints.title') }}</h5>
+        <b-form-checkbox
+          v-model="fieldConstraintExists"
+          class="mt-3"
+          @change="toggleFieldConstraint"
+        >
+          {{ $t('constraints.description') }}
+        </b-form-checkbox>
+      </div>
+
+      <div class="mt-4">
+        <b-row>
+          <b-col
+            cols="12"
+            sm="6"
+          >
+            <b-form-group
+              :label="$t('constraints.valueModifiers')"
+            >
+              <b-form-select
+                v-model="constraint.modifier"
+                :options="modifierOptions"
+              >
+                <template #first>
+                  <b-form-select-option
+                    :value="undefined"
+                    disabled
+                  >
+                    {{ $t('constraints.none') }}
+                  </b-form-select-option>
+                </template>
+              </b-form-select>
+            </b-form-group>
+          </b-col>
+          <b-col
+            cols="12"
+            sm="6"
+          >
+            <b-form-group
+              :label="$t('constraints.multiValues')"
+            >
+              <b-form-select
+                v-model="constraint.multiValue"
+                :options="multiValueOptions"
+                :disabled="!field.isMulti"
+              >
+                <template #first>
+                  <b-form-select-option
+                    :value="undefined"
```

Comment:
Should there be a none option? if you think about it you can check one way or another. Same for the one in unique values. Except for if the field is not multi, then it should say the same as it does in the mockups!